lib/libutils/isoc/bget_malloc.c

```diff
@@ -339,20 +324,21 @@ static void *raw_calloc(size_t hdr_size, size_t ftr_size, size_t pl_nmemb,
 		size_t pl_size)
 {
 	size_t s = hdr_size + ftr_size + pl_nmemb * pl_size;
-	void *ptr;
+	void *ptr = NULL;
 
 	raw_malloc_validate_pools();
 
 	/* Check wrapping */
 	if (s < pl_nmemb || s < pl_size)
```

Comment:
That will not work either, `pl_nmemb * pl_size` is likely to wrap also. GCC has a builtins for this, `__builtin_mul_overflow()` etc.